contracts/src/v0.8/operatorforwarder/dev/LinkTokenReceiver.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+// solhint-disable custom-errors
+abstract contract LinkTokenReceiver {
+  // @notice Called when LINK is sent to the contract via `transferAndCall`
+  // @dev The data payload's first 2 words will be overwritten by the `sender` and `amount`
+  // values to ensure correctness. Calls oracleRequest.
+  // @param sender Address of the sender
+  // @param amount Amount of LINK sent (specified in wei)
+  // @param data Payload of the transaction
+  function onTokenTransfer(
+    address sender,
+    uint256 amount,
+    bytes memory data
+  ) public validateFromLINK permittedFunctionsForLINK(data) {
+    assembly {
+      // solhint-disable-next-line avoid-low-level-calls
+      mstore(add(data, 36), sender) // ensure correct sender is passed
+      // solhint-disable-next-line avoid-low-level-calls
+      mstore(add(data, 68), amount) // ensure correct amount is passed0.8.19
```

Comment:
added the pragma at the end of this comment

Reply:
fixed